backend/src/main/java/com/huellapositiva/application/controller/ESALApiController.java

```diff
@@ -99,6 +104,52 @@ public void registerESAL(@Validated @RequestBody ESALRequestDto dto,
         }
     }
 
+    @Operation(
+            summary = "Updates the information of an ESAL",
+            description = "Updates the information of the ESAL linked to the logged employee. Roles allowed CONTACT_PERSON and CONTACT_PERSON_NOT_CONFIRMED.",
+            tags = "ESAL",
+            parameters = {
+                    @Parameter(name = "X-XSRF-TOKEN", in = ParameterIn.HEADER, required = true, example = "a6f5086d-af6b-464f-988b-7a604e46062b", description = "For take this value, open your inspector code on your browser, and take the value of the cookie with the name 'XSRF-TOKEN'. Example: a6f5086d-af6b-464f-988b-7a604e46062b"),
```

Comment:
CSRF token is not passed as a header, but as a query parameter. See Postman collection in the project for usage

Reply:
Done